HomeWorkLogger/Logger/Logger.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.ObjectModel;
+using System.Text;
+
+namespace Logger;
+
+public sealed class Logger
+{
+    private static readonly Logger instance = new();
+
+    private readonly string _logsDir = ".\\logs\\";
+    private readonly string _fileExtension = ".txt";
+    private Dictionary<Level, Collection<string>> Logs = new();
```

Comment:
Also you can create special model that will contains errorText and LogLevel. You will be able to use List<ThisModel> and use linq to sort this collection. But this is a good decision as well